applications/keycloak/chart/templates/servicemonitor-realm.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.keycloak.metrics.enabled  }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "keycloak.fullname" . }}-realm
+  namespace: {{ default .Release.Namespace .Values.keycloak.metrics.serviceMonitor.namespace }}
```

Comment:
Why is a namespace configured? We do let this handle the argo-app in all other resources